src/ontology/mondo-edit.obo

```diff
@@ -480062,25 +480061,22 @@ is_a: MONDO:0019701 {source="Orphanet:99645", source="icd11.foundation:206703263
 
 [Term]
 id: MONDO:0020474
-name: cheirospondyloenchondromatosis
-def: "Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
+name: obsolete cheirospondyloenchondromatosis
+def: "OBSOLETE. Cheirospondyloenchondromatosis is an extremely rare type of enchondromatosis of very early onset (from neonatal period to infancy) characterized by symmetrical multiple enchondromas with metacarpal and phalangeal involvement resulting in short hands and feet, platyspondyly, mild to moderate short stature and intellectual disability." [Orphanet:99647]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -
```

Comment:
you can remove this comment. Or update it accordingly (eg if there is a term to consider)